lib/index.d.ts

```diff
@@ -1,3 +1,3 @@
-import type { Syntax } from 'postcss';
-const postcssHtml: Syntax;
+import type { Syntax } from "postcss";
+const postcssHtml: (options?: Record<string, any>) => Syntax;
```

Comment:
It is also a function and has Syntax properties. Could you please define a more correct type?

Reply:
done